ocaml/idl/api_errors.ml

```diff
@@ -302,10 +302,12 @@ let pool_joining_host_cannot_have_vms_with_current_operations = "JOINING_HOST_CA
 let pool_joining_host_cannot_be_master_of_other_hosts = "JOINING_HOST_CANNOT_BE_MASTER_OF_OTHER_HOSTS"
 let pool_joining_host_connection_failed = "JOINING_HOST_CONNECTION_FAILED"
 let pool_joining_host_service_failed = "JOINING_HOST_SERVICE_FAILED"
-let pool_joining_host_must_have_physical_management_nic = "POOL_JOINING_HOST_MUST_HAVE_PHYSICAL_MANAGEMENT_NIC"
```

Comment:
We can't really remove error codes, because this may break existing clients. We need to keep the ones we have on trunk currently (even if they become unused), and just add the new ones alongside.